README.md

```diff
@@ -155,6 +155,14 @@ az containerapp update \
   --image $(ko build ./cmd/app)
 ```
 
+Or [Oracle Cloud Functions](https://www.oracle.com/cloud-native/functions/)
```

Comment:
This section should go below the `* Note` on line 166, which is related to the ACR functions example above this.

Also, if there are any restrictions that Oracle Cloud Functions imposes, such as a required base image or registry, that would be useful to spell out in a similar `* Note` section.

Reply:
Thanks. Updated.